### ISSUE TITLE
Getting warning ` should be ` (backtick should be backtick)

### ISSUE THREAD

**Issue by bnfrnz** [palantir/tslint#4402]
### Bug Report

- __TSLint version__: 5.12.0
- __TypeScript version__: 2.6.2
- __Running TSLint via__:  CLI (via ionic-app-scripts)

#### TypeScript code being linted

```ts
this.toastCtrl.present(`Please tick "Yes, I'm sure" to confirm`);
```
Note that all three ` and " and ' are being used for the string.

with `tslint.json` configuration:

```json
"quotemark": [true, "single", "avoid-escape", "avoid-template"]
```

#### Actual behavior
Getting warning: \` should be \`
Note that both of these are backticks. So it asks me to replace a backtick with a backtick. This makes no sense to me. Either I'm missing something here or this is a bug.

#### Expected behavior
Expected no warning. Never got a warning. This just started appearing I after updated tslint yesterday.

**Comment by bnfrnz** [comment]
I tried StackBlitz but can't seem to get tslint to work. Not sure if I'm too thick but as soon as I add tslint the repo stops working. Which other cloud tool is good to share the minimal project?

This is my local vscode:

![screen shot 2019-01-10 at 13 35 16](https://user-images.githubusercontent.com/26401091/50948215-aa0f4980-14dc-11e9-8912-963692ff2673.png)

Same warning appears in cli...

**Comment by adidahiya** [comment]
ah, I was able to repro the issue in the tslint-playground: https://palantir.github.io/tslint-playground/?saved=N4Igxg9gJgpiBcID0SAEBlALgQwE6dUwE8AHASwDsBzVSwgCxlUllQCMIAPQiVKmAgGcc+GFAB0AHQrTMEbMIDCmXABtxJXDEEwKmABQADAAqqYCppjJgA1qkkgAmtoA0qAJIByALapBAVy0HHmYICgAzMlxvQwBKAG5pEBdwMMiqBGQ0Rwh-ZmwKVG9oMnCiBiZICLIqQOwrMNRGLVQAdzJMelQiXNxUCFaKKQoUVABZbBsmAJbO+u7e1AArQUayQVQAN2xVMig3AqhacIW83H9CyjlpUYhOmD71gNdCXHKwRltKGnvQilWzKhwhA+g9cCDBMNpMBpKg4fYQDBOJhdFBBA54KgANoOTCCXZ6eBaSDebyosQOAC6Llh8Ic5zM6IQqBhhXh7IcAEd-HcYN48DYMdiVP4YG4HIJvmYHOKQNhNhA9gBabRgbAkGAyhHyxVQJUo7wkVT1TUgSm0uEAX2k1ooIEtQA

I'll take another look...
